### ISSUE TITLE
Hammerspoon: Ctrl/Esc key

### ISSUE THREAD

**Issue by qbedard** [timbedard/dotfiles#9]
I should be able to, on the laptop keyboard only, hold the Caps Lock key for Ctrl and tap it for Esc.

**Comment by qbedard** [comment]
[Here is an implementation](https://github.com/jasonrudolph/keyboard/commit/01a7a5bd8a1e521756d1ec34769119ead5eee0b3), but it doesn't appear to distinguish which keyboard its input is coming from.
